code/modules/mob/living/carbon/life.dm

```diff
@@ -31,5 +31,8 @@
 
 		. = 1
 
-		if(!client && !mind && species)
-			species.handle_npc(src)
+		// If we have an AI, and we're in a fugue state or without client.
+		// Give control to AI.
+		if(istype(ai) && ((stat & FUGUE) || (!client && !mind && species)))
+			if(ai.can_process())
+				ai.do_process()
```

Comment:
This appears to be copy-paste of https://github.com/NebulaSS13/Nebula/pull/552/files#diff-2de2df982739a4fbdb54a4b2715d6f92R84-R88

Reply:
It is. Should be fine.

Reply:
Dunno why carbon/life has a separate life that doesn't call its predecessor life, but I didn't write the code and I have no desire to understand why that change was made. If it's not broke don't fix it.